src/raft/recv_install_snapshot.c

```diff
@@ -9,6 +11,9 @@
 #include "replication.h"
 
 #include "../lib/sm.h"
+#include "src/raft.h"
+#include "src/raft/recv_install_snapshot.h"
+#include "src/raft/uv_os.h"
```

Comment:
I think the relative paths here are wrong, they should be `../raft.h` etc. (Arguably we should switch to just passing `-Isrc` into the build so we can dispense with all the `..` stuff but for now that's where we're at.)